src/tests/ftest/util/test_utils_pool.py

```diff
@@ -147,6 +147,10 @@ def __init__(self, context, dmg_command, cb_handler=None,
         self.quantity = BasicParameter(None, 1)
         self.min_targets = BasicParameter(None, 1)
 
+        # Parameter to control setting and unsetting of log mask
```

Comment:
We shouldn't introduce more spelling errors
```
wrong-spelling-in-comment, Wrong spelling of a word 'unsetting' in a comment:
```
We should either use a different word or add this one to `utils/cq/words.dict`

Reply:
Done. Thanks